src/views/self/view_weapons.cpp

```diff
@@ -280,7 +284,7 @@ namespace big
 			ImGui::PushItemWidth(250);
 			if (ImGui::BeginListBox("Saved Loadouts", ImVec2(200, 200)))
 			{
-				for (auto filename : persist_weapons::list_weapon_loadouts())
+				for (std::string filename : persist_weapons::list_weapon_loadouts())
```

Comment:
Any reason for changing the type from `auto` to `std::string`?

Reply:
nope, just don't like auto